docs/styles/proselint/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+  "author": "jdkato",
+  "description": "A Vale-compatible implementation of the proselint linter.",
+  "email": "support@errata.ai",
+  "lang": "en",
+  "url": "https://github.com/errata-ai/proselint/releases/latest/download/proselint.zip",
+  "feed": "https://github.com/errata-ai/proselint/releases.atom",
+  "issues": "https://github.com/errata-ai/proselint/issues/new",
+  "license": "BSD-3-Clause",
+  "name": "proselint",
```

Comment:
Why include this folder in git? The readme is not useful and I hope there are no custom changes, so this repository should only know of which version it's pinned to?

Reply:
I was on the fence about including the whole proselint folder. Before you can use vale with plugins you have to invoke `vale sync` and I wasn't sure that would work with the docker version. I guess I should stop being lazy and just try it.